src/main/java/net/sf/jabref/model/entry/Date.java

```diff
@@ -0,0 +1,62 @@
+package net.sf.jabref.model.entry;
+
+import java.time.DateTimeException;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+import java.time.temporal.TemporalAccessor;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Optional;
+
+public class Date {
```

Comment:
Sure you want to call that Date? Will create a lot of confusion with the import hell..